src/atopile/cli/build.py

```diff
@@ -27,71 +26,40 @@
 
 @click.command()
 @project_options
-@click.option("--debug/--no-debug", default=None)
 @muffle_fatalities
-def build(config: Config, debug: bool):
+def build(build_ctxs: list[BuildContext]):
     """
     Build the specified --target(s) or the targets specified by the build config.
```

Comment:
Would be nice to be able to build a specific target. For example `ato build default`.